__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const request = require("supertest");
+const app = require("../app.js");
+const seed = require("../db/seeds/seed.js");
+const {
+  topicData,
+  userData,
+  articleData,
+  commentData,
+} = require("../db/data/test-data");
+const db = require("../db/connection.js");
+
+beforeEach(() => {
+  return seed({ topicData, userData, articleData, commentData });
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe.only("GET /api/topics", () => {
```

Comment:
Please remember to remove any only's on the test suite before pushing as this will meen husky isn't doing its job :)